templates/app/env

```diff
@@ -4,5 +4,6 @@ FOREST_ENV_SECRET=<%= forestEnvSecret %>
 <% if (!ssl) {%>#<% } %>SSL_DATABASE=true
 <% if (!encrypt) {%>#<% } %>ENCRYPT_DATABASE=true
 <% if (forestUrl) {%>FOREST_URL=<%= forestUrl %><% } %>
+<% if (devRenderingId) {%>DEV_RENDERING_ID=<%= devRenderingId %><% } %>
 <% if (dbSchema) {%>DATABASE_SCHEMA=<%= dbSchema %><% } %>
-<% if (port) {%>PORT=<%= port %><% } %>
\ No newline at end of file
+<% if (port) {%>PORT=<%= port %><% } %>
```

Comment:
We could add FOREST_URL here in case we detect the admin server to be localhost to make dev faster, just a thought

Reply:
Let's do this on another PR I think